vividus-tests/src/main/resources/story/integration/HttpRequest.story

```diff
@@ -25,3 +25,9 @@ Then a JSON element by the JSON path '$.cookies' is equal to '{"vividus-cookie":
 Scenario: Verify HTTP cookies are cleared
 When I send HTTP GET to the relative URL '/cookies'
 Then a JSON element by the JSON path '$.cookies' is equal to '{}'
+
+Scenario: I wait for response code $responseCode for $duration duration retrying $retryTimes times $stepsToExecute
+When I wait for response code `200` for `PT10S` duration retrying 10 times
+|step                                                                                                                                                     |
+|When I issue a HTTP GET request for a resource with the URL '${vividus-test-site-url}/api/delayed-response?clientId=de2f4058-1a72-40dc-a6c7-ad37c6c07249'|
```

Comment:
UUID should be generated at runtime, if hardcoded one is used and tests are triggered from several branches, there is a risk of collision
Vividus expression will help to generate the value: `#{generate(Internet.uuid)}`